project/src/com/google/daggerqueryui/scripts/analyze_input_field_content.js

```diff
@@ -116,18 +124,32 @@ const queryExecutor = (function() {
 
 $("#query-input").on('keyup', function (event) {
   const query = $(this).val().trim().split(' ');
-  const queryName = query[0];
+  const queryName = query[0].toLowerCase();
 
   const queryNameElement = $(this).closest('.interactive-input-container').find('.query-name');
   if (!$(this).validateQueryName(queryName)) {
     queryNameElement.hide();
+
+    // The user can specify the node name without the query name.
+    // If such a node exists in the graph, it will be drawn.
+    if (event.key === 'Enter' && query.length === 1) {
+      queryExecutor.processQuery([$.EXISTS_QUERY_NAME, query[0]], false);
+    }
+
     return;
   }
 
   // Highlights query's name when it is valid.
   queryNameElement.html(queryName).show();
 
-  if (event.key === 'Enter' && $(this).validateParameters(query)) {
-    queryExecutor.processQuery(query, true);
+  if (event.key === 'Enter') {
+    if ($(this).validateParameters(query)) {
+      queryExecutor.processQuery(query, true);
+    } else {
+      $(this).markInputFieldAsInvalid(
+        `The number of passed parameters for ${queryName} query is incorrect.` +
+        ` Expected: ${$.supportedQueries.get(queryName)}, got: ${query.length - 1}.`
```

Comment:
I'm worried this error message is fragile if the implementation of `validateParameters` is enhanced later. How about throwing this error from within the `validateParameters` function and catch it here, or change the function name to `validateNumParameters`?

Reply:
Let's [throw an exception](https://github.com/googleinterns/dagger-query/pull/50/commits/0729f1e537903c176f8a99207319220bd9239596) and hide the details and reason in `validateParameters`.

✅